### PR TITLE
Jayne marketplace

### PR DESCRIPTION
**Pull request by jaynecn**
##  What does this PR do
Formats coaches into their own cards
Cards are responsive
Each coach has a link to their own Apppointed booking widget

##  Description of task to be completed
Users should be able to see a list of coaches

##  How should this manually be tested
https://dev-coach-staging.herokuapp.com/marketplace

##  What are the relevant Trello board stories
As a user I want to be able to see all coaches

##  Questions (If any):

### CODE REVIEW

**Comment by jaynecn** [comment]
[As a user I want to be able to see all coaches](https://trello.com/c/TeIRREtl/43-as-a-user-i-want-to-be-able-to-see-all-coaches)

File: src/components/Marketplace.js [hunk 1]
```diff
@@ -1,55 +1,89 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (domeccleston) [inline src/components/Marketplace.js]: If we're using Redux, we want that to be our single source of truth for our app so we should avoid using useState.

Comment (domeccleston) [inline src/components/Marketplace.js]: If we want to use Redux, that should be our single source of truth so we should avoid useState

Comment (jaynecn) [reply src/components/Marketplace.js]: Now refactored with redux

**Pull request merged**